DataFormats/CaloRecHit/interface/CaloCluster.h

```diff
@@ -36,6 +36,8 @@ namespace reco {
 
     // super-cluster flags
     enum SCFlags { cleanOnly = 0, common = 100, uncleanOnly = 200 };
+    // hcal cluster flags (used for pf)
+    enum HCalFlags { badHcalMarker = 1 };
```

Comment:
This I would leave as is because one shouldn't be using the two sets of flags together (on hcal clusters one should use only the hcal flags, on superclusters only the sc flags)